### PR TITLE
Release notes for Forms/Deploy 12

### PR DESCRIPTION
**Pull request by AndyButland**
Includes:

- Documented breaking changes from removing obsolete code constructs for Forms 12
- Documented breaking changes from removing obsolete code constructs for Deploy 12

Can be reviewed and go live any time on or after May 24th.

### CODE REVIEW

File: 12/umbraco-deploy/upgrades/version-specific.md [hunk 1]
```diff
@@ -10,6 +10,36 @@ This page covers specific upgrade documentation for when migrating to a new majo
 If you are upgrading to a minor or patch of Deploy you can find the details about the changes in the [release notes](/11/umbraco-deploy/release-notes.md) article.
 {% endhint %}
 
+<details>
+
+<summary>Version 12</summary>
+
+Version 11 of Umbraco Deploy has a minimum dependency on Umbraco CMS core of `12.0.0`. It runs on .NET 7.
```
Comment (jonat123) [inline 12/umbraco-deploy/upgrades/version-specific.md:17]:
```suggestion
Version 12 of Umbraco Deploy has a minimum dependency on Umbraco CMS core of `12.0.0`. It runs on .NET 7.
```

File: 12/umbraco-deploy/upgrades/version-specific.md [hunk 2]
```diff
@@ -10,6 +10,36 @@ This page covers specific upgrade documentation for when migrating to a new majo
...
+
+The forms deployment component has a minimum dependency on Umbraco Forms of `12.0.0`.
+
+### Breaking changes
+
+Version 11 contains a number of breaking changes. We don't expect many projects to be affected by them as they are in areas that are not typical extension points.  For reference though, the full details are listed here:
```
Comment (jonat123) [inline 12/umbraco-deploy/upgrades/version-specific.md:23]:
```suggestion
Version 12 contains a number of breaking changes. We don't expect many projects to be affected by them as they are in areas that are not typical extension points.  For reference though, the full details are listed here:
```

File: 12/umbraco-forms/installation/version-specific.md [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+
+<details>
+
+<summary>Version 12</summary>
+
+Version 12 of Umbraco Forms has a minimum dependency on Umbraco CMS core of `12.0.0`. It runs on .NET 7.
+
+#### Breaking changes
+
+Version 11 contains a number of breaking changes. If you do run into any, they should be straightforward to adjust and recompile.
```
Comment (jonat123) [inline 12/umbraco-forms/installation/version-specific.md:17]:
```suggestion
Version 12 contains a number of breaking changes. If you do run into any, they should be straightforward to adjust and recompile.
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@AndyButland Looks good!

Only noticed that there were a few times in the v12 parts it said v11, I have made suggestions for them :D 

Once fixed I'll make sure it is merged in on the 24th.

**Comment by AndyButland** [comment]
Yes, 23rd please if that's OK.  There was some confusion on the date but we're looking to get these out tomorrow.

**Comment by jonat123** [comment]
@AndyButland As the space is not live yet(Meaning if you don't have access to gitbook, you can't see it yet) I will merge this PR in.

Then we will make sure that it is all public tomorrow with the release of the RC :)

**Pull request merged**